pkg/testdata/models/apis/ec2/0000-00-00/generator.yaml

```diff
@@ -68,8 +68,11 @@ operations:
     output_wrapper_field_path: VpcEndpoint
 
 resources:
-  DHCPOptions:
-
+  DhcpOptions:
+    fields:
+      DHCPConfigurations.Values:
+        set:
+          - from: AttributeValue.Value
```

Comment:
If you add:

`    method: CREATE`

to the above `generator.yaml` file, the test will fail because you are passing `OpTypeCreate` in the test but `OpTypeList` is hard-coded in the SetResourceForStruct method...

Reply:
Good point; will update depending on convo above